File: webservices/CORE/Ciudad.php
<?php


class Ciudad
{
    public $idciudad;
    public $nombre;
    public $idprovincia;
    public $estado;

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }


    /**
     * Ciudad constructor.
     * @param $idciudad
     * @param $nombre
     * @param $idprovincia
     */
    public function __construct($idciudad, $nombre, $idprovincia,$estado)
    {
        $this->idciudad = $idciudad;
        $this->nombre = $nombre;
        $this->idprovincia = $idprovincia;
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getIdciudad()
    {
        return $this->idciudad;
    }

    /**
     * @param mixed $idciudad
     */
    public function setIdciudad($idciudad)
    {
        $this->idciudad = $idciudad;
    }

    /**
     * @return mixed
     */
    public function getNombre()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $nombre
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    /**
     * @return mixed
     */
    public function getIdprovincia()
    {
        return $this->idprovincia;
    }

    /**
     * @param mixed $idprovincia
     */
    public function setIdprovincia($idprovincia)
    {
        $this->idprovincia = $idprovincia;
    }



}

File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Ciudad.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Ciudad implements Serializable {
    private int idciudad ;
    private String nombre;
    private int idprovincia;
    private int id_base;

    public Ciudad() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Ciudad(int idciudad, String nombre, int idprovincia) {
        this.idciudad = idciudad;
        this.nombre = nombre;
        this.idprovincia = idprovincia;
    }

    public int getIdciudad() {
        return idciudad;
    }

    public void setIdciudad(int idciudad) {
        this.idciudad = idciudad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getIdprovincia() {
        return idprovincia;
    }

    public void setIdprovincia(int idprovincia) {
        this.idprovincia = idprovincia;
    }
}
